packages/react/src/exampleTypes/ExpandableTagsQuery/index.js

```diff
@@ -55,16 +161,26 @@ let TagsWrapper = observer(
     sanitizeTags = true,
     splitCommas = true,
     maxTags = 1000,
+    generationsCollapse,
+    enableKeywordGenerations,
     ...props
   }) => {
     let TagWithPopover = React.memo(
       observer((props) => {
-        let result = _.get(['context', 'results', props.value], node)
+        let result = _.get(['context', 'tags', props.value], node)
+        let generations = _.get(
+          ['context', 'keywordGenerations', props.value],
+          node
+        )
         let tagProps = {
           ...props,
-          ...(!_.isNil(result)
-            ? { label: `${props.value} (${toNumber(result)})` }
-            : {}),
+          ...(!_.isNil(result) && {
+            label: `${props.value} (${toNumber(result)})`,
+          }),
+          ...(_.isNil(result) &&
+            !_.isNil(generations) && {
+              label: `${props.value} (${toNumber(generations)})`,
```

Comment:
You can simplify this by cascading the two paths:
```js
let count = F.cascade(
  [`context.tags.${props.value}`, `context.keywordGenerations.${props.value}`],
  node
)
```
and then in tagProps, just have one case where you're spreading on an object with label